AppController/djinn.rb

```diff
@@ -1613,6 +1613,8 @@ def stop_app(app_name, secret)
             Djinn.log_warn("(stop_app) got exception talking to " +
               "zookeeper: #{e.message}.")
           end
+          # Prevent future deploys from using the local app archive.
+          FileUtils.rm_rf("#{PERSISTENT_MOUNT_POINT}/apps/#{app_name}.tar.gz")
```

Comment:
is there a need to remove also the /var/apps/#{app_name} part? or this is taken care independently?